instrumentation/grape/README.md

```diff
@@ -0,0 +1,90 @@
+# OpenTelemetry Grape Instrumentation
+
+The Grape instrumentation is a community-maintained instrumentation for [Grape][grape], a REST-like API framework for Ruby.
+
+It relies on the Grape built-in support for `ActiveSupport::Notifications` (more info [here](https://github.com/ruby-grape/grape#active-support-instrumentation)).
+
+It currently supports the following events:
+
+- `endpoint_run.grape`
+- `endpoint_render.grape`
+- `endpoint_run_filters.grape`
+- `format_response.grape`
```

Comment:
`format_response` is not part of the DDog instrumentation but we decided to add it because we felt it could add value with formatting errors (we've had some experiences where this would have been useful).

However, it is important to point out that this event is not a child of `endpoint_run`, so we'll be relying on another parent span (e.g. from Rack or Sinatra) to connect it to the trace. In other words, while `endpoint_render` and `endpoint_run_filters` have `endpoint_run` as their parent span, `format_response` does not.

Because of this, we're open to removing it if you feel it would cause trouble 😄